src/common/helpers/github-url.js

```diff
@@ -1,6 +1,16 @@
-const getGitHubRepoUrl = (owner, name) => {
+import parseGitHubUrl from 'parse-github-url';
+
+export const getGitHubRepoUrl = (owner, name) => {
   let repository = !name ? owner : `${owner}/${name}`;
   return `https://github.com/${repository}`;
 };
 
-export default getGitHubRepoUrl;
+export const parseGitHubRepoUrl = (url) => {
+  const repo = parseGitHubUrl(url);
+  return (repo && repo.repo) ? {
+    owner: repo.owner,
+    name: repo.name,
+    fullName: repo.repo,
+    url: url
+  } : null;
```

Comment:
If our repository naming scheme is as close as it is to parse-github-url's, is it worth just using their naming scheme instead?